yarn.lock

```diff
@@ -1768,6 +1768,11 @@
   resolved "https://registry.yarnpkg.com/@discoveryjs/json-ext/-/json-ext-0.5.7.tgz#1d572bfbbe14b7704e0ba0f39b74815b84870d70"
   integrity sha512-dBVuXR082gk3jsFp7Rd/JI4kytwGHecnCoTtXFb7DB6CNHp4rg5k1bhg0nWdLGLnOV71lmDzGQaLMy8iPLY0pw==
 
+"@ecl/theme-dev@3.11.1":
```

Comment:
i noticed this..this is because we didn't clean up the package.json of the preset-dev package, which still has all the deprecated components as dependecies. More than doing this, i would completely remove also the preset-dev, the css is not used, the js is still being used but i'd say we can accept to duplicate the exports in each preset and get also rid of this "legacy" preset.